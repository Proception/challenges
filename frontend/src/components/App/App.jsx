import React, { Component } from 'react';
import {
  BrowserRouter as Router
} from 'react-router-dom';
import './App.scss';
import 'bootstrap/dist/css/bootstrap.min.css';

import ConsentView from '../../views/ConsentView.jsx';


export default class App extends Component {

  render() {
    return (
      <Router>
        <ConsentView />
      </Router>
    );
  }
}
