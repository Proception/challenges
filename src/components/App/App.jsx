import React, { Component } from 'react';
import {
  BrowserRouter as Router
} from 'react-router-dom';
import ConsentView from '../../views/ConsentView.jsx';

import './App.scss';
import 'bootstrap/dist/css/bootstrap.min.css';


export default class App extends Component {

  render() {
    return (
      <Router>
        <ConsentView defaultMenu={'consentForm'} />
      </Router>
    );
  }
}
