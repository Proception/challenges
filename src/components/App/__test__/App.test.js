import React from 'react';
import renderer from 'react-test-renderer';

import { Provider } from 'react-redux';
import configureStore from '../../../store';
import App from '../App.jsx';


test('App component', () => {
  const component = renderer.create(
    <Provider store={configureStore()}>
      <App />
    </Provider>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
