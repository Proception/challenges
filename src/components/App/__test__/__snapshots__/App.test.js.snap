// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-sm-12 col-md-3 col-lg-3 col-xl-3"
    >
      <div
        className="side-menu-card-container"
      >
        <div
          class="side-menu-card"
        >
          <div
            className="side-menu-content"
          >
            <div
              className="side-menu-list"
            >
              <a
                className="side-menu-item-link"
                onClick={[Function]}
              >
                <div
                  className="side-menu-item-active"
                >
                  <div
                    className="side-menu-item-image"
                  />
                  <div
                    className="text-space"
                  >
                    Give consent
                  </div>
                </div>
              </a>
              <a
                className="side-menu-item-link"
                onClick={[Function]}
              >
                <div
                  className="side-menu-item"
                >
                  <div
                    className="side-menu-item-image"
                  />
                  <div
                    className="text-space"
                  >
                    Collected consents
                  </div>
                </div>
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="col-sm-12 col-md-9 col-lg-9 col-xl-9"
    >
      <div
        class="side-menu-card"
      >
        get consent from user
      </div>
    </div>
  </div>
</div>
`;
